articles/encrypting-gdrive-using-rclone/index.md

```diff
@@ -0,0 +1,226 @@
+author: louise-findlay
+date: 2020-08-25T00:00:00-10:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/encrypting-gdrive-using-rclone/hero.png
+    alt: rclone gdrive header image
+---
+This is a how-to guide on setting up Rclone to create an encrypted folder where users can store data on Google Drive which will be encrypted before upload.
+Readers will understand how to install Rclone, add a Google Drive remote, obtain a Google Client ID, how to manage files on the local filesystem and Google Drive (copy, move, sync, and mount) and create an encrypted remote.
+<!--more-->
+
+### Table of Contents
+* [Introduction](#introduction)
+* [Installing Rclone](#installing-rclone)
+* [Creating Your First Rclone Remote](#creating-your-first-rclone-remote)
+  * [Create Google Drive Client ID](#create-google-drive-client-id)
+  * [Continuing Rclone Configuration](#continuing-rclone-configuration)
+  * [Allow Rclone Access to Your Google Drive](#allow-rclone-access-to-your-google-drive)
+  * [Finish Remote Setup and Test](#finish-remote-setup-and-test)
+* [Uploading Files to A Remote (Copying, Moving & Syncing)](#uploading-files-to-a-remote-copying-moving-and-syncing)
+  * [Copy/Move/Sync Files](#copy-move-sync-files)
+  * [Check the Files Have Transferred](#check-the-files-have-transferred)
+* [Mount A Google Drive Folder Using Rclone](#mount-a-google-drive-folder-using-rclone)
+  * [Creating the Rclone Mount](#creating-the-rclone-mount)
+  * [Testing the Rclone Mount](#testing-the-rclone-mount)
+* [Encrypt Your Google Drive Remote with Rclone](#encrypt-your-google-drive-remote-with-rclone)
+  * [Create A Crypt Remote](#create-a-crypt-remote)
+  * [Upload Files to the Crypt Remote](#upload-files-to-the-crypt-remote)
+  * [Check Files Have Been Uploaded to the Encrypted Remote](#check-files-have-been-uploaded-to-the-encrypted-remote)
+* [Uninstalling Rclone](#uninstalling-rclone)
+
+### Introduction
+Rclone is a cross-platform command-line tool that connects remotes (cloud storage providers) with the local filesystem. Similar to [rsync](https://en.wikipedia.org/wiki/Rsync) but for cloud storage, this tool is vital for remotes where some operating systems aren’t natively supported and it provides advanced functionality such as syncing and mounting.
+
+### Installing Rclone
+First, we need to download Rclone. Using the command below, this will automatically install the latest version of Rclone and the one that is correct for your distribution and architecture of your machine.
+
+```bash
+curl https://rclone.org/install.sh | sudo bash
+```
+
+There are security implications on installing Rclone this way because the script could be malicious but this way is far simpler because you don’t have to figure out the architecture your computer is running on, download the correct version, unzip the file and move it to the correct location.
```

Comment:
This is a great warning. Perhaps we could also link to the rclone install doc in case rclonse switch to a better install practice, or if readers want to understand it more first. I.e. https://rclone.org/install/

Reply:
Thanks. I've added a link to the install docs so readers can learn about alternative installation methods.